apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -575,7 +586,7 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                  * as optional because not all children have hint/description
                  * popups. {@see modules > forms > CheckboxChild}
                  */
-                const grant: { label: string; value: string; hint?: any; disabled?: boolean } = {
+                const grant: { label: any; value: string; hint?: any; disabled?: boolean } = {
```

Comment:
Can't we create an interface `{ label: any; value: string; hint?: any; disabled?: boolean }`? And I think it is better to use `ReactElement` instead of `any`.

Reply:
fixed with 4b42bf24627f9471b59beda4d795f5d2147dd68e